ev3dev/sound.py

```diff
@@ -44,11 +44,9 @@ def _make_scales(notes):
     return res
 
 
-class Sound:
+class Sound(object):
```

Comment:
Why do we want to do this? Can't you already instantiate the class if desired?

Reply:
For consistency with all of our other classes.